doc/ref_arch/openstack/chapters/chapter02.md

```diff
@@ -58,46 +58,46 @@ Traceability to Reference Model.
-| `req.inf.stg.02` | Storage | The Architecture **should** provide Object storage for VM Instances. | |
-| `req.inf.stg.03` | Storage | The Architecture **may** provide a file system service (file system storage solution) for VM Instances. | |
-| `req.inf.stg.04` | Storage | The Architecture **may** support Software Defined Storage (SDS) that seamlessly supports shared block storage, object storage and flat files. | |
-| `req.inf.stg.05` | Storage | The Architecture **should** be able to accommodate VNFs that store back into its image through use of hypervisor attached volumes. | |
-| `req.inf.stg.06` | Storage | The Architecture **should** make the immutable images available via location independent means. | |
-| `req.inf.stg.07` | Storage | The Architecture **should** provide high-performance and horizontally scalable VM storage. | |
+| `req.inf.com.04` | Compute | The Architecture **must** be able to support multiple CPU SKU options to support various infrastructure profiles (Base, Network Intensive, and Compute Intensive).| [RA-1 4.4.1. "Support for Profiles and T-shirt instance types"](https://github.com/cntt-n/CNTT/blob/master/doc/ref_arch/openstack/chapters/chapter04.md#441-support-for-profiles-and-t-shirt-instance-types) |
+| `req.inf.com.05` | Compute | The Architecture **must** support Hardware Platforms with NUMA capabilities.| [RA-1 4.4.1. "Support for Profiles and T-shirt instance types"](https://github.com/cntt-n/CNTT/blob/master/doc/ref_arch/openstack/chapters/chapter04.md#441-support-for-profiles-and-t-shirt-instance-types) |
+| `req.inf.com.06` | Compute | The Architecture **must** support CPU Pinning of the vCPUs of VM instance.| [RA-1 4.4.1. "Support for Profiles and T-shirt instance types"](https://github.com/cntt-n/CNTT/blob/master/doc/ref_arch/openstack/chapters/chapter04.md#441-support-for-profiles-and-t-shirt-instance-types) |
+| `req.inf.com.07` | Compute | The Architecture **must** support different hardware configurations to support various infrastructure profiles (Base, Network Intensive, and Compute Intensive).| [RA-1 3.3.3. "Host aggregates providing resource pooling"](https://github.com/cntt-n/CNTT/blob/master/doc/ref_arch/openstack/chapters/chapter03.md#333-host-aggregates-providing-resource-pooling) |
+| `req.inf.com.08` | Compute | The Architecture **must** support allocating certain number of host cores/threads to non-tenant workloads such as for OpenStack services. | Please see ["CPU Pinning" section](https://docs.openstack.org/nova/latest/user/flavors.html) |
+| `req.inf.com.09` | Compute | The Architecture **must** ensure that the host cores/threads assigned to a workload are thread-sibling aware: that is, that a core and its associated SMT threads are either all assigned to non-tenant workloads or all assigned to tenant workloads. | Please see ["CPU Pinning" section](https://docs.openstack.org/nova/latest/user/flavors.html) |
```

Comment:
Isn't `req.inf.com.08` and `req.inf.com.09` more about isolating CPUs for services running on compute vs CPUs consumed by VMs? Do not understand how CPU Pinning covers it.

Reply:
@deebhatia Thanks for catching that -- wrong reference.